metro.config.js

```diff
@@ -4,5 +4,8 @@ const { getDefaultConfig } = require('expo/metro-config');
 const defaultConfig = getDefaultConfig(__dirname);
 
 defaultConfig.resolver.assetExts.push('cjs');
+// this will be the default in the future, so let's be future-proof
+defaultConfig.resolver.enablePackageExports = true;
```

Comment:
this is a major change in how metro resolves packages. It's not needed but it's going to be the default in future, so I figured I'd give it a try